src/models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
```

Comment:
What is the requirement for this file?

Reply:
I used sequelize-cli which generated this file when i initiated the migration

Reply:
See if you can remove this file.

Reply:
@udaygaidhane30 can this file be removed?

Reply:
So sorry @ananth243 , didn't saw your comment, looking into it now